dev/source/docs/style-guide.rst

```diff
@@ -402,22 +402,23 @@ Functions that return a single physical value or variables that represent a phys
     uint16 get_angle() { ... };
     float distance;
 
-Functions or variables that represent a value relative to a frame should be suffixed with the frame first, then with the physical unit (if any).
+Functions or variables that represent a value relative to a frame should be suffixed with the frame first, then with the physical unit (if different than SI units). SI suffixes are occasionally used to avoid confusion in modules with similar variables of different units,but only if necessary to avoid confusion.
```

Comment:
```suggestion
Functions or variables that represent a value relative to a frame should be suffixed with the frame first, then with the physical unit (if different than SI units). SI suffixes are occasionally used to avoid confusion in modules with similar variables of different units, but only if necessary to avoid confusion.
```

(space added after comma)

Reply:
fixed